pydicom/pixel_data_handlers/rle_handler.py

```diff
@@ -374,6 +374,29 @@ def _rle_decode_segment(data):
 
 
 # RLE encoding functions
+def _wrap_rle_encode_frame(src: bytes, **kwargs) -> bytes:
+    """Wrapper for use with the encoder interface.
+
+    .. versionadded:: 2.2
+
+    Parameters
+    ----------
+    src : bytes
+        A single frame of image data to be RLE encoded.
+    **kwargs
+        Optional parameters for the encoding function.
+
+    Returns
+    -------
+    bytes
+        An RLE encoded frame.
+    """
+    bytes_allocated = kwargs['bits_allocated'] // 8
```

Comment:
I plan to rewrite the RLE handler in another PR so its not array -> bytes -> array. And also to return little endian ordered data.